frontend/src/v5/ui/controls/inputs/colorPicker/colorPicker.component.tsx

```diff
@@ -49,7 +49,7 @@ export const ColorPicker = ({ value: inputValue, defaultValue, onChange }: Color
 		<ActionMenu
 			onOpen={() => setSelected(true)}
 			onClose={() => setSelected(false)}
-			TriggerButton={<ColorPickerPreview selected={selected} color={value?.color} />}
+			TriggerButton={<ColorPickerPreview selected={selected} color={value?.color} disabled={disabled} />}
```

Comment:
If a user is not an admin, `ColorPicker` is disabled, but shouldn't a user with no permission be prohibited to add/edit a group in the first place?

Reply:
Although it does make me wonder how the disabled color picker should work. Should a non-admin user be able to open up the interface to see the hex value and opacity?

Reply:
I didn't know about containers and federations until now lol.
The problem though is that (at least in this branch) the most useful piece of information is the description since everything else can be inferred/read from the rendered group item. However, the description uses a single line input, meaning if it breaks over several lines, the user would still not be able to read it. @george-hadfield what do you think about this?